clubs/README.md

```diff
@@ -638,3 +641,21 @@ Teachers/club sponsors/school administrators will sometimes try to ban a club fr
 > When we call ourselves "Hack Club", it means we're building apps, websites, games, and anything else you can imagine.
 
 If that still doesn't work, just reach out to us on [Slack](https://hackclub.slack.com/messages/C0J7C6YET/). A bunch of club leaders have had to deal with this and we're happy to help out.
+
+### What if a member of my club does something inappropriate?
+
+You should warn and ask them stop. You'll want to make sure to tell them what is inappropriate about their actions, as they themselves might not realize that are doing it.
+
+If this persists past the warning, then talk to your teacher/club sponsor/school administrator about the proper course of action to take.
+
+Refer to our [conducts page](https://github.com/hackclub/hackclub/blob/master/CONDUCT.md) for more information. 
+
+### What if our adviser does not want to let us meet every week?
+
+Sometimes, an advisor might not have the time in their schedule to meet weekly. Ask if you can get a second advisor to meet on the weeks that your current one can't.
+
+Alternatively, you could find a public space to meet in and have your club meeting there.
+
+### How do I transition leadership roles to another person?
```

Comment:
I'm partial to dropping this from the FAQ. I don't think many club leaders think much about how to transition their club leadership?